substrate/frame/proxy/src/lib.rs

```diff
@@ -663,8 +667,17 @@ pub mod pallet {
 			proxy_type: T::ProxyType,
 			disambiguation_index: u16,
 		},
+		PureKilled {
+			pure: T::AccountId,
+			who: T::AccountId,
```

Comment:
```suggestion
			spawner: T::AccountId,
```

Also some documentation would be nice.

Reply:
I think it should also have the `index`, otherwise it is difficult to know